src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/Ordering/PasteOrdering.cs

```diff
@@ -18,15 +20,15 @@ internal class PasteOrdering : IPasteHandler, IPasteDataObjectProcessor
     {
         public const int OrderPrecedence = 10000;
 
-        private readonly ConfiguredProject _configuredProject;
+        private readonly ActiveConfiguredProject<ConfiguredProject> _configuredProject;
```

Comment:
Can you update the commit message to explain how this is different from what we did before?

Reply:
Added some longer-form explanation.